src/v2/cookbook/serverless-blog.md

```diff
@@ -115,18 +115,17 @@ Then create `components/BlogHome.vue` which will be your blog homepage that list
     }
   }
 </script>
-Display the result
```

Comment:
The text "Display the result" seems meaningless so I removed it.

Reply:
Agreed